src/lesson6_task1/LightTransport.java

```diff
@@ -0,0 +1,56 @@
+package lesson6_task1;
+
+
+import java.nio.charset.StandardCharsets;
+import java.util.Scanner;
+
+public class LightTransport extends LandTransport {
+    String bodyworkType;
+    int countPassengers;
+    float way;
+
+    //конструктор
+    public LightTransport(int powerHours, int maxSpeed, String brand, float weight, int countOfWheels, float fuelConsumption,
+                          String bodyworkType, int countPassengers) {
+        super(powerHours, maxSpeed, brand, weight, countOfWheels, fuelConsumption);
+        this.bodyworkType = bodyworkType;
+        this.countPassengers = countPassengers;
+        this.way = way;
+    }
+
+    //возвращает строку с описанием всех характеристик объекта
+    @Override
+    public String getDescription() {
+        this.technicalChar = "Марка " + this.brand + ":\n-Мощность " + this.powerHours + " лошадиных сил;\n-Мощность " + getPowerKw() + " кВ;\n" +
+                "-Максимальная скорость " + this.maxSpeed + " км/ч;\n-Масса " + this.weight + " кг;\n" +
+                "-Количество колёс " + this.countOfWheels + ";\n-Расход топлива " + this.fuelConsumption + " л/100км;\n" +
+                "-Тип кузова - " + this.bodyworkType + ";\n-Кол-во пассажиров " + this.countPassengers + " человек.\n";
+        return this.technicalChar;
```

Comment:
@Nadya101  Можно укоротить и избежать лишнего создания новой переменной
```suggestion
        return "Марка " + this.brand + ":\n-Мощность " + this.powerHours + " лошадиных сил;\n-Мощность " + getPowerKw() + " кВ;\n" +
                "-Максимальная скорость " + this.maxSpeed + " км/ч;\n-Масса " + this.weight + " кг;\n" +
                "-Количество колёс " + this.countOfWheels + ";\n-Расход топлива " + this.fuelConsumption + " л/100км;\n" +
                "-Тип кузова - " + this.bodyworkType + ";\n-Кол-во пассажиров " + this.countPassengers + " человек.\n";
```

Reply:
Исправила, странно почему я так сразу не сделала &#128528;